clang/test/SemaSYCL/initiation_interval.cpp

```diff
@@ -27,7 +29,7 @@
 [[intel::initiation_interval(1)]] void func6(); // expected-note {{previous attribute is here}}
 [[intel::initiation_interval(3)]] void func6(); // expected-warning {{attribute 'initiation_interval' is already applied with different arguments}}
 
-// Tests for Intel FPGA loop fusion function attributes compatibility
+// Tests for Intel FPGA initiation_interval and disable_loop_pipelining function attributes compatibility.
```

Comment:
Also good changes but unrelated to the patch.

Reply:
Done